src/connection/index.ts

```diff
@@ -150,10 +154,10 @@ const coinbaseWalletConnection: Connection = {
   },
 }
 
-export function getConnections() {
+export function getConnections(isDarkMode: boolean) {
```

Comment:
I think its better to move isDarkMode into getIcon, and remove the need for hooks altogether (we can refactor back to having just getConnection / getConnections instead of having useGetConnections)

Reply:
sounds good @cartcrom , done